tests/system/test_integration.py

```diff
@@ -143,33 +145,37 @@ def count_library_frames(self, doc, lf):
 
 class SourcemappingIntegrationTest(ElasticTest, ClientSideBaseTest):
 
+    ignore_override_warnings = ["WARN.*Overriding sourcemap"]
+
     @unittest.skipUnless(INTEGRATION_TESTS, "integration test")
     def test_backend_error(self):
         path = 'http://localhost:8000/test/e2e/general-usecase/bundle.js.map'
-        r = self.upload_sourcemap(file_name='bundle.js.map', bundle_filepath=path)
+        r = self.upload_sourcemap(
+            file_name='bundle.js.map', bundle_filepath=path)
         assert r.status_code == 202, r.status_code
         self.wait_for_sourcemaps()
 
         self.load_docs_with_template(self.get_error_payload_path(),
                                      'http://localhost:8200/v1/errors',
                                      'error',
                                      1)
-        self.assert_no_logged_warnings()
+        self.assert_no_logged_warnings(self.ignore_override_warnings)
```

Comment:
I don't think that this is right, there should not be a warning issued, as the sourcemap is uploaded only once, so please don't ignore it here.